main.go

```diff
@@ -342,6 +342,15 @@ var clientHelp = `
     the credentials inside the server's --authfile. defaults to the
     AUTH environment variable.
 
+    --socks5-auth, An optional username and password for socks5 server
+    in the form: "<user>:<pass>". These credentials will then be required
+    by Socks5 server when new client is connecting. Defaults to the
+    AUTH_SOCKS5 environment variable.
+
+    --socks5-allowed-hosts, An optional list of hosts (comma delimited) on
+    the client side which will be allowed to connect to by socks5 client.
+    Only FQDN values are accepted
+
```

Comment:
i think this needs to be in the common section, since the socks server can run in both client and server modes